src/tests/Workleap.Extensions.OpenAPI.Analyzers.Tests/BaseAnalyzerTest.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Testing;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Testing;
+using Microsoft.CodeAnalysis.Testing.Verifiers;
+using Workleap.Extensions.OpenAPI.TypedResult;
+
+namespace Workleap.Extensions.OpenAPI.Analyzers.Tests;
+
+public class BaseAnalyzerTest<TAnalyzer> : CSharpAnalyzerTest<TAnalyzer, XUnitVerifier>
+    where TAnalyzer : DiagnosticAnalyzer, new()
+{
+    private const string CSharp10GlobalUsing = """
```

Comment:
The constant name doesn't match the value. `Swashbuckle.AspNetCore.Annotations` is not part of C# 10.